app/controllers/api/v0/listings_controller.rb

```diff
@@ -10,7 +10,6 @@ class ListingsController < ApiController
       before_action :authenticate_with_api_key_or_current_user!, only: %i[create update]
       before_action :authenticate_with_api_key_or_current_user, only: %i[show]
       before_action :set_cache_control_headers, only: %i[index show]
-      before_action :set_listing, only: %i[update]
```

Comment:
This is no longer needed here as the concern now uses an `included` hook to set up this action.